src/context/CaptureDetailContext.js

```diff
@@ -36,15 +41,45 @@ export function CaptureDetailProvider(props) {
 
   // EVENT HANDLERS
 
-  const getCaptureDetail = async (id) => {
-    if (id == null) {
-      log.debug('getCapture called with no id');
-      return Promise.resolve(STATE_EMPTY.capture);
-    } else {
-      return api.getCaptureById(id).then((capture) => {
-        setState({ ...state, capture });
-        return capture;
-      });
+  const getCaptureDetail = async (capture) => {
+    log.debug('getCaptureDetail:', {
+      reference_id: capture.reference_id,
+      id: capture.id,
+    });
+
+    // getOrganization gets the wrong org when this is in the global context
+    const BASE_URL = {
+      LEGACY: `${API_ROOT}/api/${getOrganization()}trees/`,
+      CAPTURE_MATCH: `${TREETRACKER_API}/captures?reference_id=`,
+    };
```

Comment:
This can be used to determine which microservice to make the request to   in the future, but we'll have to pass a second argument to indicate which tool initiated the request.

Seem like a solution?

Reply:
Looks good to me.